publish/HandleInertiaMiddleware.php

```diff
@@ -25,12 +22,7 @@ public function __construct(
     public function share(ServerRequestInterface $request): array
     {
         return array_merge(parent::share($request), [
-            'auth' => [
-                'user' => fn () => $this->coreAuthenticatableProvider->user(),
-            ],
-            'flash' => [
-                'alert' => fn () => $this->session->get('alert'),
-            ],
```

Comment:
These handlers are required in order to keep inertia works properly. CoreAuthenticableProvider is defined in https://github.com/onix-systems-php/hyperf-core package. We either need to:
1. add core package as dependency
2. redo this predefined handlers to put them into config or return them from separated protected method that can be redefined by the end user